### PR TITLE
feat: image component

### PR DESCRIPTION
**Pull request by adith-a-danthi**
Added image component
- Responsive Image
- Square, Rounded, Circle Image

image

### CODE REVIEW

File: docs/components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  object-fit: cover;
+}
+
+.img-rounded {
+  border-radius: 10px;
+  object-fit: cover;
+}
+
+.img-circle {
+  border-radius: 9999px;
```
Comment (ZuberDunge) [inline docs/components/image/image.css:13]: you can just give border radius 50%

**Review by ZuberDunge — COMMENTED** [review]
looks good to me

File: docs/components/image/image.css (hunk 1 above)
Comment (adith-a-danthi) [reply docs/components/image/image.css:13]: The outcome is different if the width and height are not equal, hence I used `9999px`

**Review by sameep-s — APPROVED** [review]
Looks good. you can go on and merge

**Pull request merged**